test/nl/surf/eduhub_rio_mapper/smoke_test.clj

```diff
@@ -0,0 +1,82 @@
+(ns nl.surf.eduhub-rio-mapper.smoke-test
+  (:require
+    [clj-http.client :as client]
+    [clojure.edn :as edn]
+    [clojure.java.io :as io]
+    [clojure.test :refer :all]
+    [nl.surf.eduhub-rio-mapper.cli :as cli]
+    [nl.surf.eduhub-rio-mapper.clients-info :as clients-info]
+    [nl.surf.eduhub-rio-mapper.job :as job]
+    [nl.surf.eduhub-rio-mapper.ooapi :as ooapi]
+    [nl.surf.eduhub-rio-mapper.rio :as rio])
+  (:import [java.io PushbackReader]))
+
+(defn- ls [dir-name]
+  (map #(.getName %) (.listFiles (io/file dir-name))))
+
+(defn only-one-if-any [list]
+  (assert (< (count list) 2) (prn-str list))
+  (first list))
+
+(defn- numbered-file [basedir nr]
+  {:post [(some? %)]}
+  (str basedir
+       "/"
+       (->> basedir
+            (ls)
+            (filter #(.startsWith % (str nr "-")))
+            (only-one-if-any))))
+
+(defn- make-playbacker [idx]
+  (let [count-atom (atom 0)
+        dir        (numbered-file "test/fixtures/smoke" idx)]
+    (fn [actual-request]
+      (let [i     (swap! count-atom inc)
+            fname (numbered-file dir i)]
+        (assert fname (str "No recorded request found for:" (pr-str actual-request)))
```

Comment:
Maak hier maar een exception van. `(throw (ex-info ...))` want dit is gewoon een error.